src/lib/project.ts

```diff
@@ -2,28 +2,16 @@ import type { Commit, CommitHeader, CommitsHistory } from "@app/lib/commit";
 import type { Host } from "@app/lib/api";
 import type { ProjectResult } from "@app/lib/search";
 
+import * as utils from "@app/lib/utils";
 import { Request } from "@app/lib/api";
-import { Seed, defaultSeedPort } from "@app/lib/seed";
-import { isFulfilled, isOid, isRadicleId } from "@app/lib/utils";
+import { Seed } from "@app/lib/seed";
+import { isFulfilled, isOid, isRepositoryId } from "@app/lib/utils";
 
 export type Id = string;
```

Comment:
Can we also remove this alias?
Since at one place it is used as a Repository Id and at the other as Node Id, it's confusing.

Reply:
cb3451dce4a3a0f0f96c4e218b62ea82371ace39